Dangerfile

```diff
@@ -1,15 +1,15 @@
 message "@#{github.pr_author} Thank you for your pull request! I'm here to help with some tips and recommendations. Please take a look at the list above and help us review and accept your contribution!"
```

Comment:
@jywarren Since message will now show sequentially you may want to put this statement at the bottom or change this message to `...look at the list below..`